web/src/composebox_typeahead.ts

```diff
@@ -1246,10 +1256,14 @@ export function content_typeahead_selected(
                 beginning = beginning.slice(0, syntax_start_index) + "#**" + stream_name + "** ";
                 break;
             }
-            beginning =
-                beginning.slice(0, syntax_start_index) +
-                topic_link_util.get_stream_topic_link_syntax(stream_name, item.topic) +
-                " ";
+            const stream_topic_link_syntax =
+                item.topic_display_name === realm.realm_empty_topic_display_name
```

Comment:
> item.topic_display_name === realm.realm_empty_topic_display_name

This is incorrect.

Just to be clear:
`realm.realm_empty_topic_display_name` and `util.get_final_topic_display_name("");` is not the same thing. There's translation involved.